clang/lib/3C/DeclRewriter.cpp

```diff
@@ -729,6 +747,13 @@ void FunctionDeclBuilder::buildDeclVar(const FVComponentVariable *CV,
     ABRewriter.getBoundsString(CV->getExternal(), Decl,
                                !getExistingIType(CV->getExternal()).empty());
 
+  if(CV->getExternal()->isGenericChanged()) {
+    Type = CV->getExternal()->mkString(Info.getConstraints());
+    if (BoundsStr != "")
```

Comment:
Why the `BoundsStr` check? For a wrapper around `malloc`, for example, the wrapper will have a bounds string.

Reply:
I added it because the existing code dealt with it. I'm not sure what you mean by the malloc wrapper having a bounds string.